src/AdminConsole/appsettings.Development.json

```diff
@@ -11,9 +11,15 @@
     "CustomerAuthPlanName": "Professional Test"    
   },
   "ConnectionStrings": {
-    "sqlite": "Data Source=Data/adminconsole_dev.db"
+    "sqlite": "Data Source=adminconsole_dev.db"
   },
   "Passwordless": {
-    "ApiUrl": "https://localhost:8001"
+    "ApiUrl": "http://localhost:7001",
+    "ApiSecret": "TestValue",
+    "ApiKey": "TestValue"
```

Comment:
Thought: could we use proper values that we seed in #30?

Reply:
Are you seeding values in #30? I don't see such a change.

Reply:
Not seeding in #30 currently, but if we're adding database creation / migration there, we could also add seeding